baseplate/sidecars/live_data_watcher.py

```diff
@@ -9,7 +10,11 @@
 from pathlib import Path
 from typing import Any
 from typing import NoReturn
+from typing import Optional
 
+import boto3  # type: ignore
```

Comment:
why do we need `type: ignore` here?

Reply:
Lint was complaining for some reason....

baseplate/sidecars/live_data_watcher.py:15: error: Skipping analyzing "boto3": found module but no type hints or library stubs